src/modules/mavlink/streams/OPEN_DRONE_ID_SYSTEM.hpp

```diff
@@ -68,13 +80,14 @@ class MavlinkStreamOpenDroneIdSystem : public MavlinkStream
 		sensor_gps_s vehicle_gps_position;
 		home_position_s home_position;
 
-		if (_vehicle_gps_position_sub.update(&vehicle_gps_position) && _home_position_sub.copy(&home_position)) {
-			if (vehicle_gps_position.fix_type >= 3
-			    && home_position.valid_alt && home_position.valid_hpos) {
+		if (_vehicle_gps_position_sub.update(&vehicle_gps_position) &&
+		    _home_position_sub.copy(&home_position)) {
+			if (vehicle_gps_position.fix_type >= 3 && home_position.valid_alt &&
+			    home_position.valid_hpos) {
 
 				mavlink_open_drone_id_system_t msg{};
 				msg.target_component = 0; // 0 for broadcast
-				msg.target_system = 0; // 0 for broadcast
+				msg.target_system = 0;    // 0 for broadcast
 				// msg.id_or_mac // Only used for drone ID data received from other UAs.
 				msg.operator_location_type = MAV_ODID_OPERATOR_LOCATION_TYPE_TAKEOFF;
```

Comment:
From [this comment](https://github.com/PX4/PX4-user_guide/issues/2605#issuecomment-1619004431):

> Also I need to check https://github.com/PX4/PX4-Autopilot/pull/21647, but according to FAA regulation, operator location needs to be fixed or dynamic. Value take-off position is only valid for Remote ID broadcast modules.

Reply:
Yeah this is a major pain point in FAA rules. The DIY side of the house will only want fixed or takeoff location so as to avoid being Karen'd via Dronescanner and they are the builders that put together a kit with a pixhawk, so I would assume they would be fine under the broadcast module exception? The vendors will want to CYA so that's why they want to go with live operator position.

Reply:
It's a purely legal questions, so I'd be asking `le0r` on [discord](https://discord.com/channels/1022170275984457759/1038284900081614879/1128976116040871967), and possibly @friissoren.

From a technical standpoint the FAA rule is "confused" because it assumes  that there is an operator/GCS - there might not be. The fact is that the vehicle only knows for sure its own position and position of home - no other information is guaranteed. 

My take is that we should use operator position if available, and fall back to home position (or last-seen in current flight otherwise). Does FC know when it only has a broadcast module - or would it be up to the remote ID module to send that it is not-ready if it is not a broadcast module and does not have operator position?

Reply:
> It's a purely legal questions, so I'd be asking `le0r` on [discord](https://discord.com/channels/1022170275984457759/1038284900081614879/1128976116040871967), and possibly @friissoren.
> 
> From a technical standpoint the FAA rule is "confused" because it assumes that there is an operator/GCS - there might not be. The fact is that the vehicle only knows for sure its own position and position of home - no other information is guaranteed.
> 
> My take is that we should use operator position if available, and fall back to home position (or last-seen in current flight otherwise). Does FC know when it only has a broadcast module - or would it be up to the remote ID module to send that it is not-ready if it is not a broadcast module and does not have operator position?

At the moment very busy, not much online on discord.  

The answer from the FAA on standard Remote ID:

"The rule states any aircraft produced for operation in the US National Airspace System after 16 Sep 2022 must be produced with standard remote ID.  Manufacturers are still allowed to sell non-remote ID aircraft produced before that date.

Effective 16 Sep 2023, any aircraft that is not equipped with standard remote ID may comply with the rule by 1) using a broadcast-only module, or 2) operating exclusively within the geographic boundaries of a FAA-Recognized Identification Area."

The F3586 rule mandates the following on Operator position (section 5.1.3.6)

"For Standard Remote ID, the Operator Location/Altitude Type in the System Message shall be either “1. Dynamic” and use a system that accurately reports the location of the ground control station (GCS); or “2. Fixed” where the location is automatically programmed into the UAS. The GCS location must correspond with the location of the operator."

So I don't see any fallback regarding the live operator position. If you don't have you will fail the F3586 
 tests,. (One of those tests is the live operator position.) And drones produced after 16 Sep 2023 require a standard Remote ID solution.